govcd/vapp.go

```diff
@@ -43,6 +43,7 @@ type VappNetworkSettings struct {
 	DNSSuffix            string
 	GuestVLANAllowed     bool
 	StaticIPRanges       []*types.IPRange
+	AddDHCPService       bool
```

Comment:
1. What is the use case of having `AddDHCPService` set to `false`?
2. Doesn't vApp network always have a DHCP service? It may be not configured, but isn't it always there?
3. We had issue with DHCP being not enabled which you fixed. Does it still work correctly with this PR? https://github.com/terraform-providers/terraform-provider-vcd/issues/78

Reply:
1. We need this due GO limitations with default property initialization and we can't check if they are nil. So not send half empty xml DHCP service configuration I need explicitly state that DHCP info was added and marshalling should include these properties. 
2. According responses I got three states - not configured, configured disabled, configured enabled
3. This code part isn't used with edge gtw

Reply:
>     1. we can't check if they are nil.
Please elaborate this point.

Reply:
We can't identify when value wasn't provided or when it is init value for e.g. default int value will be 0 or user will provide value 0.

Reply:
Why the DHCP properties can't be like `DHCPIPRange *types.IPRange`, which can be nil ?

Reply:
If you think it's better I could change struct to more levels and then it should work also.

Reply:
It is not a matter of personal preferences here. In this library we use this approach several times. Using a `struct` pointer simplifies the marshaling, and you don't need to have an explicit `AddDHCPService`

Reply:
Nice suggestion!

Reply:
Changed code.